pkg/models/devops/devops.go

```diff
@@ -544,8 +545,8 @@ func ToJson(req *http.Request) ([]byte, error) {
 }
 
 func GetNotifyCommit(req *http.Request) ([]byte, error) {
-	baseUrl := fmt.Sprintf(jenkins.Server + GetNotifyCommitUrl + req.URL.RawQuery)
-	log.Infof("Jenkins-url: " + baseUrl)
+	baseUrl := fmt.Sprint(jenkins.Server + GetNotifyCommitUrl + req.URL.RawQuery)
```

Comment:
why use fmt.Sprint here?

Reply:
here the var req.URL.RawQuery mabey have some special characters such as "%s", but not should be formart.

Reply:
Is there a problem with using "+" directly?

Reply:
It's ok in test.